File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.cpp
class Solution {
public:
    int subtractProductAndSum(int n) {
        int num = 0;
        int sum_mult = 1;
        int sum_add = 0;
        while (n > 0){
            num = n%10;
            n /= 10;
            sum_mult *= num;
            sum_add += num;
        }
        num = sum_mult - sum_add;
        return num;
    }
};

File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.py
class Solution:
    def subtractProductAndSum(self, n: int) -> int:
        addsum = 0
        multsum = 1
        while n > 0:
            num = n % 10
            n = n // 10
            addsum += num
            multsum *= num
        return multsum -addsum